docs/diagrams/testing-component-diagram.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
```

Comment:
I like that diagram, but it has little to do with the testing approach described below. I would move that to another section e.g. component diagram etc.

Reply:
It's a fair point. We only used it to be explicit about our assumptions at this stage. Only after we imagined the components were we able to actually think about how they would be tested.

We should eventually remove this diagram. Maybe once #17 is specified?

Reply:
I agree that this diagram should live somewhere else but could be referenced here! Given that it was included in https://github.com/get10101/10101/pull/33 I think we can remove it from this PR.